### DISCUSSION TITLE
Welcome to space-case Discussions!

### DISCUSSION THREAD

**Discussion by CircleTect** [CircleTect/space-case#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sasat228** [comment]
Greetings. I’m very surprised that such a gorgeous project didn’t get a big response from the audience. Although I am the owner of a low-quality, but large 3D printer (print area 40x40x45cm), I really appreciate projects in which optimization and simplification of production for the most basic user plays an important role. I have already printed several free projects, now my hands have reached your body! I have already bought aluminum (yes, this will be a budget assembly) tubes, now in the process of printing all the parts. except for one thing - this is the power supply! somewhere under the video or on the sff forum it was mentioned that it is planned to install a regular ATX power supply in addition to SFX... Are there any developments in this direction? I don’t specifically cut aluminum tubes yet, I’m ready to sacrifice 2-3 cm of width or height for the opportunity to install an atx psu. I will be glad to hear your answer, I hope the notification reaches you!
